### DISCUSSION TITLE
Is the 'dedit' pseudonym for 'deepin-editor' gone ?

### DISCUSSION THREAD

**Discussion by aiamuzz** [linuxdeepin/developer-center#2981] [Q&A | 问答板块]
Hi,

I just noticed `dedit` the pseudonym for `deepin-editor` is no longer available ... is it gone for good ?

so do we have to use `sudo deepin-editor /root/file_name` going forward ?

**Comment by BLumia** [comment]
Hi, you can still create such an alias by yourself if needed, assume you are using bash, then go to your `~/.bashrc` file and add your preferred alias :)
